realm/realm-jni/src/io_realm_internal_Util.cpp

```diff
@@ -78,7 +80,9 @@ JNIEXPORT jlong JNICALL Java_io_realm_internal_Util_nativeGetMemUsage(JNIEnv*, j
 JNIEXPORT jstring JNICALL Java_io_realm_internal_Util_nativeGetTablePrefix(
     JNIEnv* env, jclass)
 {
-    return to_jstring(env, string(TABLE_PREFIX));
+    string table_prefix(TABLE_PREFIX);
```

Comment:
There's a common pattern of creating a stack `std::string table_prefix` to wrap the compile-time constant `TABLE_PREFIX`. Could this be a file-level `static`, or at least locally a `static`, or would you prefer to leave the compiler to optimise that (which I'm a bit doubtful about)?